Parsers/ASimRegistryEvent/test/Windows_Event_VimRegistryEvent_DataTest.csv

```diff
@@ -0,0 +1,11 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [EventVendor] of type [Enumerated]: [""Microsoft""] (Schema:RegistryEvent)"
+"(0) Error: 2 invalid value(s) (up to 10 listed) in 4 records (40.0%) for field [EventType] of type [Enumerated]: [""RegistryKeyNotify"",""Other""] (Schema:RegistryEvent)"
```

Comment:
and this

Reply:
done

Reply:
@Alekhya0824 this is pending

Reply:
updated eventVendor but eventType haven't found

Reply:
eventvendor added but eventtpe haven't added because in documentation it haven't mentioned supported type